src/planscape/plan/views.py

```diff
@@ -225,7 +225,7 @@ def list_plans_by_owner(request: HttpRequest) -> HttpResponse:
                  .annotate(projects=Count('project', distinct=True))
                  .annotate(scenarios=Count('project__scenario')))
         return JsonResponse(
-            [_serialize_plan(plan, False) for plan in plans],
+            [_serialize_plan(plan, True) for plan in plans],
```

Comment:
Is there any reason for this update in plan?

Reply:
yep we need geography on `list_plans_by_owner` to get total acres

Reply:
Hm, since 'plan' is getting replaced with 'planning', we should be getting this information from list_planning_areas in planning/views.py instead

Reply:
oh you're right! I'll revert this change, this was when I was using the other endpoint, thanks!